metricbeat/magefile.go

```diff
@@ -93,6 +94,27 @@ func Update() error {
 	return sh.Run("make", "update")
 }
 
+// MockedTests runs the HTTP tests using the mocked data inside each {module}/{metricset}/testdata folder.
+// Use MODULE={module_name} to run only mocked tests with a single module
+// Use GENERATE=true or GENERATE=1 to regenerate JSON files
+func MockedTests() error {
+	goTestDefaultArgs := mage.DefaultGoTestUnitArgs()
+
+	goTestDefaultArgs.ExtraFlags = []string{"github.com/elastic/beats/metricbeat/mb/testing/data/."}
```

Comment:
There's a `Package` field in `GoTestArgs` type but the order of the executed contents isn't correct when using it. By placing everything as `ExtraFlag` I could control the order of the flags and package run.

Reply:
Probably @andrewkroh knows best here.